src/Instance/InstanceDetails/SubInstanceList/SubInstanceList.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Link } from 'react-router-dom';
+
+import {
+  MultiColumnList,
+  KeyValue,
+} from '@folio/stripes/components';
+
+import { getIdentifiers } from '../../../utils';
+import { indentifierTypeNames } from '../../../constants';
+import useReferenceData from '../../../hooks/useReferenceData';
+
+const SubInstanceList = ({
+  titles,
+  id,
+  titleKey,
+  label,
+}) => {
+  const {
+    ISSN,
+    ISBN,
+  } = indentifierTypeNames;
+  const {
+    identifierTypesById,
+  } = useReferenceData();
+
+  const formatter = {
+    title: row => (row[titleKey] ?
+      <Link
+        to={`/inventory/view/${row[titleKey]}`}
+      >
+        {row.title}
+      </Link> :
+      row.title || '-'),
+    hrid: row => row.hrid || '-',
+    publisher: row => row.publication?.[0]?.publisher ?? '-',
+    publisherDate: row => row.publication?.[0]?.dateOfPublication ?? '-',
+    issn: row => getIdentifiers(row.identifiers, ISSN, identifierTypesById) || '-',
+    isbn: row => getIdentifiers(row.identifiers, ISBN, identifierTypesById) || '-',
```

Comment:
I'm not totally sure but maybe in terms of `accessibility` it is better to use `noValue` from constants.js instead of '-'?

Reply:
@SergiyVSergiyenko great idea. I was not aware of that!

Reply:
We also have `<NoValue>` in stripes-components so I think I will go with that.